recipes/10/10_over.md

```diff
@@ -0,0 +1,53 @@
+# 10 Rolling Aggregations on Time Series Data
+
+:bulb: This example will show how to aggregate multiple rows, which are related to the current row, via an `OVER` window. A typical use case are rolling aggregations.
```

Comment:
This sounds a bit overcomplicated. What about:

"This example will show how to calculate an aggregate or cumulative value based on a group of rows using an `OVER` window."